src/Razor/src/Microsoft.VisualStudio.DevKit.Razor/Microsoft.VisualStudio.DevKit.Razor.csproj

```diff
@@ -1,40 +1,34 @@
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <Description>Razor is a markup syntax for adding server-side logic to web pages. This package contains the language server assets for C# DevKit.</Description>
     <EnableApiCheck>false</EnableApiCheck>
-    <IsShippingPackage>false</IsShippingPackage>
-    <RemoveDevicePlatformSupport>true</RemoveDevicePlatformSupport>
-  </PropertyGroup>
-
-  <!--
-    In a vertical build, we'll only publish for the RID of the vertical.
-    In a non-vertical build, we'll publish for all RIDs of the OS we are building on (to reduce the number of CI jobs).
-  -->
-  <PropertyGroup Condition="'$(DotNetBuild)' != 'true'">
-    <RuntimeIdentifiers Condition="$([MSBuild]::IsOSPlatform('Windows'))">win-x64;win-arm64</RuntimeIdentifiers>
-    <RuntimeIdentifiers Condition="$([MSBuild]::IsOSPlatform('Linux'))">linux-x64;linux-musl-x64;linux-arm64;linux-musl-arm64</RuntimeIdentifiers>
-    <RuntimeIdentifiers Condition="$([MSBuild]::IsOSPlatform('OSX'))">osx-x64;osx-arm64</RuntimeIdentifiers>
-  </PropertyGroup>
+    <IsShippingPackage>true</IsShippingPackage>
+    <IsPackable>true</IsPackable>
 
-  <PropertyGroup Condition="'$(DotNetBuild)' == 'true'">
-    <RuntimeIdentifiers>$(TargetRid)</RuntimeIdentifiers>
-    <RuntimeIdentifier>$(TargetRid)</RuntimeIdentifier>
-    <SelfContained>false</SelfContained>
-    <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
-    <RidSpecificPublishNoBuildProperty>NoBuild=true</RidSpecificPublishNoBuildProperty>
+    <!-- This is not a standard nuget package and only consumed by the extension build. We don't care if the folder structure doesn't match what nuget expects. -->
+    <NoWarn>$(NoWarn);NU5100</NoWarn>
   </PropertyGroup>
 
   <ItemGroup>
     <Compile Include="..\Microsoft.VisualStudio.LanguageServices.Razor\Telemetry\TelemetryReporter.cs" Link="Telemetry\TelemetryReporter.cs" />
   </ItemGroup>
 
+    <!--
+    Additional runtime dependencies that we must include in the NuGet package as they are not a part of the main language server package.
+    Do not remove GeneratePathProperty="True" as it is required below to add the package to the package we generate.
+    -->
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.Common" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" />
-    <PackageReference Include="Microsoft.VisualStudio.Telemetry" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Common" GeneratePathProperty="True" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" GeneratePathProperty="True" />
+    <PackageReference Include="Microsoft.VisualStudio.Telemetry" GeneratePathProperty="True" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="$(PkgMicrosoft_VisualStudio_Telemetry)\lib\netstandard2.0\Microsoft.VisualStudio.Telemetry.dll" Pack="true" PackagePath="content" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
Why do we need to include `MS.VS.Telemetry.dll` as content? Isn't the `<PackageReference>` enough?

Reply:
It might be tbh. Devkit in roslyn does this, but that might be leftover from before it being a nuget package.

Reply:
Updated, we should be fine with this.